gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1237,7 +1293,7 @@ runMatcher(ofx_info* info, char * selected_filename, gboolean go_to_next_file)
         gnc_gen_trans_list_show_all (info->gnc_ofx_importer_gui);
         
         // Show or hide the check box for reconciling after match, depending on whether a statement was received.
-        gnc_gen_trans_list_show_reconcile_after_close_button (info->gnc_ofx_importer_gui, info->statement != NULL, info->run_reconcile);
+        gnc_gen_trans_list_show_reconcile_after_close_button (info->gnc_ofx_importer_gui, g_list_length (info->statement), info->run_reconcile);
```

Comment:
ditto... best leave `info->statement != NULL` alone... see https://developer.gimp.org/api/2.0/glib/glib-Doubly-Linked-Lists.html-- "There is no function to create a GList. NULL is considered to be the empty list so you simply set a GList* to NULL."